fiftyone/utils/cvat.py

```diff
@@ -3312,6 +3338,52 @@ def delete_tasks(self, task_ids):
             for task_id in pb(list(task_ids)):
                 self.delete_task(task_id)
 
+    def create_git(self, task_id, git_repo):
+        json_data = {
+            "path": git_repo,
+            "lfs": False,
```

Comment:
Since storing zips in git can quickly get out-of-hand spacewise, I imagine users of this feature will want a `git_lfs` flag exposed so they can opt-in to using LFS if desired.

Reply:
Yes, the older zip files just seem to be overwritten by newer ones. It is also possible to export the raw .xml file by just specifying an xml path instead of a zip file path.